projects/core/src/auth/user-auth/services/auth-multisite-isolation.service.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { take } from 'rxjs/operators';
+import { BaseSiteService } from '../../../site-context/facade/base-site.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthMultisiteIsolationService {
+  protected static MULTISITE_SEPARATOR = '|';
```

Comment:
why `static`? IMHO `readonly` would suffice. ;)

Reply:
Thanks, I changed to `readonly`.